nbconvert/filters/tests/test_ansi.py

```diff
@@ -61,6 +64,7 @@ def test_ansi2latex(self):
             'hello\x1b[01;34mthere': r'hello\textcolor{ansi-blue-intense}{\textbf{there}}',
             'hello\x1b[001;34mthere': r'hello\textcolor{ansi-blue-intense}{\textbf{there}}',
             '\x1b[1mhello\x1b[33mworld\x1b[0m': r'\textbf{hello}\textcolor{ansi-yellow-intense}{\textbf{world}}',
+            'he\x1b[4mll\x1b[24mo': 'he\\underline{ll}o',
```

Comment:
Is it convenient to test the conversion of inverted mode to latex as well?

Reply:
It is a bit unwieldy, but why not: dab959638199ce562534efcdd9a9d7eb9dfc31bd